supervisor/api/backups.py

```diff
@@ -143,11 +143,26 @@ def _ensure_list(item: Any) -> list:
 class APIBackups(CoreSysAttributes):
     """Handle RESTful API for backups functions."""
 
-    def _extract_slug(self, request):
+    async def _extract_slug(self, request):
```

Comment:
I think this method should be renamed since it does (and already did before this PR) more than just get the slug from the request

Reply:
Tbh I would actually prefer we don't rename it. Or at the very least, please keep `_extract` at the beginning. We can change it to `_extract_backup_by_slug` if we prefer. But if you look at all the other files under `api`, every one that takes an id of something in the resource path has a matching `_extract_<x>` method that extracts that id and returns the resource or raises. I'd like to keep that consistency to make these easy to find.